test/utils/print/TrusteeOrganisationPrintHelperSpec.scala

```diff
@@ -126,6 +127,33 @@ class TrusteeOrganisationPrintHelperSpec extends SpecBase {
           )
         )
       }
+
+      "5mld trustee has country of residence" in {
+        val userAnswers: UserAnswers = baseAnswers
+          .set(UtrYesNoPage(index), false).success.value
+          .set(mld5.CountryOfResidenceYesNoPage(index), true).success.value
+          .set(mld5.CountryOfResidenceInTheUkYesNoPage(index), false).success.value
+          .set(mld5.CountryOfResidencePage(index), "DE").success.value
+          .set(AddressYesNoPage(index), true).success.value
+          .set(AddressUkYesNoPage(index), false).success.value
+          .set(InternationalAddressPage(index), internationalAddress).success.value
+
+        val result = helper.checkDetailsSection(userAnswers, name, index, fakeDraftId)
+
+        result mustBe AnswerSection(
+          None,
+          Seq(
+            AnswerRow("trustee.individualOrBusiness.checkYourAnswersLabel", Html("Business"), Some(TrusteeIndividualOrBusinessController.onPageLoad(index, fakeDraftId).url), name),
+            AnswerRow("trustee.organisation.name.checkYourAnswersLabel", Html("Name"), Some(NameController.onPageLoad(index, fakeDraftId).url), name),
+            AnswerRow("trustee.organisation.utrYesNo.checkYourAnswersLabel", Html("No"), Some(UtrYesNoController.onPageLoad(index, fakeDraftId).url), name),
+            AnswerRow("trustee.organisation.5mld.countryOfResidenceInTheUkYesNo.checkYourAnswersLabel", Html("No"), Some(CountryOfResidenceInTheUkYesNoController.onPageLoad(index, fakeDraftId).url), name),
```

Comment:
CountryOfResidenceYesNo answer row missing (also needed in the print helper itself)